### ISSUE TITLE
Tables w/ "headers" attribute

### ISSUE THREAD

**Issue by wodrich** [facebook/react#2516]
I am working on a project that has strict accessibility requirements.  The table I am putting some data in has to implement headers and ids attributes to meet accessibility.   See W3 techniques here
http://www.w3.org/TR/WCAG20-TECHS/H43.html

Unfortunately the "headers" attribute is being stripped out because it's not supported by React (it's not on the supported attributes list).  http://facebook.github.io/react/docs/tags-and-attributes.html#html-attributes

Can you please add support for the missing standard attribute called "headers"?
